src/castra/middleware.clj

```diff
@@ -88,45 +74,14 @@
   (if (some req body-keys) data (@clj->json req data)))
 
 (defn headers
-  [{:keys [body] :as req} head extra]
+  [{:keys [body] :as _req} head extra]
   (merge head (when-not (coll? body) extra)))
 
-(def default-timeout (* 1000 60 60 24))
-
 (defn- castra-req?
   [req]
   (and (get-in req [:headers "x-castra-csrf"])
        (= :post (:request-method req))))
 
-(defn wrap-castra-session [handler ^String key & [{:keys [timeout]}]]
```

Comment:
We used to use this middleware a long time ago, but we have since switched to using the Ring session cookie middleware directly.